.github/workflows/terraform-plan-apply.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           git fetch origin main:localmain
 
           # Gets changed json files excluding the json schema
-          export FILES=$(git diff --name-only localmain | grep .json | grep -v "trial-json-schema")
+          export FILES=$(git diff --name-only localmain | grep .json | grep -v "trial-json-schema" | grep -v "\/.*_.")
```

Comment:
same comment

Reply:
I wanted to ensure that an automated deployment doesnt look at the 'manual' definition files